### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ApolloSG**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,8 +204,10 @@ My name is Vanya, and I am glad to be part of the team.
...
 
+----------------------------------------------------------------
+## Hi everyone :)
+
 --------
 ## Hi, guys
 
-![cat-forking](https://i.imgur.com/6eoCVvc.jpg)
-
+![cat-forking](https://i.imgur.com/6eoCVvc.jpg)
```
Comment (OleksiyRudenko) [inline README.md]: Please, bring a newline back.

**Comment by OleksiyRudenko** [comment]
@ApolloSG please, let me know if you need any help.

**Comment by OleksiyRudenko** [comment]
@ApolloSG 

You may commit any changes to current branch. Then push to update the PR.

Also you will need to pull from upstream (this core repo) master and then resolve conflicts, commit changes, and push again.

Do not hesitate inquiring me whenever you require any support.

**Comment by ApolloSG** [comment]
Sorry, didn't see ur first comments. I'm gonna try to bring  a new line back in 2 hours.

**Comment by ApolloSG** [comment]
I've got a problem. My previous steps:
- `git remote add upstream https://github.com/kottans/mock-repo.git`
- `git pull upstream master`
- `git checkout upstream/master`
- `git merge master`
- `git fetch`
- edited README.md
- `git add .`
- `git commit -m "some message"`

Now I'm trying to push my local commits into origin (forked kottans/mock-repo) by using command:
`git push origin`
And getting the next lines:
`remote: Permission to kottans/mock-repo.git denied to ApolloSG.
fatal: unable to access 'https://github.com/kottans/mock-repo.git/': The requested URL returned error: 403`

Tell me pls, what am I doing wrong?

**Comment by OleksiyRudenko** [comment]
@ApolloSG Nice try :)

> 1. `git remote add upstream https://github.com/kottans/mock-repo.git`
> 2. `git pull upstream master`
> 3. `git checkout upstream/master`
> 4. `git merge master`
> 5. `git fetch`
> 6. edited README.md
> 7. `git add .`
> 8. `git commit -m "some message"`

Steps 1 and 2 already did what was required to sync with original repo.
Steps 3 and 4 is an attempt to prepare commits for uploading to upstream. That's what actually results in access error.
Step 5 most probably did actually nothing as your local history is in sync with your own origin.

**How to fix**

You need to get rid of merge commit created by steps 3 and 4 and reset upstream/master pointer to its original position. Basically, when you reset upstream/master pointer the merge commit would become untracked and would silently disappear sometime in future. It would but it will not. The reason is that you have next commit (steps 6..8) in your local master.

To fix your local history and get rid of merge commit you may want to reset master to its original position and cherry-pick your latest commit by its hash (take the note of the hash before resetting master as it will become untracked).

Will you try to complete these actions?

**Comment by ApolloSG** [comment]
I'm gonna try it

**Comment by ApolloSG** [comment]
Well, I tried but failed. My actions:
- `git checkout upstream/master`
- `git checkout HEAD^2`
getting: `Previous HEAD position was 2b51f3f Merge pull request #172 from kottans/patch-add-editorconfig
HEAD is now at 9945119 Add trailing newline :)`
- ` branch -f upstream/master 9945119`
- `log --graph --full-history --all --color --pretty=format:"%x1b[31m%h%x09%x1b[32m%d%x1b[0m%x20%s" -n10`
and getting [this](https://fex.net/134269216793?fileId=949036796)
- `git checkout master`
- ` git checkout master^`
- `git  git log --graph --full-history --all --color --pretty=format:"%x1b[31m%h%x09%x1b[32m%d%x1b[0m%x20%s" -n10'
and getting [this](https://fex.net/134269216793?fileId=949041696)
- `git branch -f master 2b51f3f`
- `git checkout master`
getting: `Warning: you are leaving 1 commit behind, not connected to
any of your branches:  9d1a51c say hi 
...
Switched to branch 'master'
Your branch is up to date with 'origin/master'.`
- `git cherry-pick f96e117`
- `git push origin`

And getting familiar message(
`remote: Permission to kottans/mock-repo.git denied to ApolloSG.
fatal: unable to access 'https://github.com/kottans/mock-repo.git/': The requested URL returned error: 403`

**Comment by OleksiyRudenko** [comment]
@ApolloSG 

`git branch -f upstream/master` did not what you expected it to do. Please, note that `git log` output shows two `upstream/master` refs. So it just created a branch named `upstream/master`.

This is fixable: `git branch -D upstream/master`.

I shall revert to you later with suggestions on how to fix.

**Comment by OleksiyRudenko** [comment]
@ApolloSG 
Do you feel you want to resolve this situation? If not just let me know, we then just close this PR and will start over from the scratch.

Otherwise, let's fix it incrementally.

### The problem

There is a branch named `upstream/master` and a real reference representing `master` at `upstream`.
The latter is not in sync with `upstream` and cannot be pushed upstream due to auth restrictions.

### Solution. Step 1.

Get rid of "fake" `upstream/master`:
`git branch -D upstream/master`

Getting references status and current tree state:
`git show-ref --head --dereference` will show all references and hashes these are referring to
`git log --graph --full-history --all --color --pretty=format:"%x1b[31m%h%x09%x1b[32m%d%x1b[0m%x20%s" -n10`

Please, post the outputs of both commands here, not screenshots please, but textual copy/paste like below:
```
$ git logg -n10
* 358deaf        (HEAD -> master, origin/master, origin/HEAD) Add Test 10 section
* 0f812b7        add ref to ServiceWorker related materials
* 71247a7        add Task 9 section
* 9997bbd        add demo screencast
* ce76dbb        fix typos in reflections headings
* b14b203        add Memory Pair Game section
* 2013ab9        bump section title
* 8228710        add course project game demo screencast
*   910c7e1      Merge branch 'master' of https://github.com/OleksiyRudenko/kottans_frontend
|\
| * 16d1b15      update TOC
```

P.S. Once you run the command below
`git config --global alias.logg "log --graph --full-history --all --color --pretty=format:\"%x1b[31m%h%x09%x1b[32m%d%x1b[0m%x20%s\""`
you will then be able to benefit from the shorthand alias e.g. `git logg` or `git logg -n10`

**Comment by ApolloSG** [comment]
@OleksiyRudenko 

Yes, I feel. 
I got rid of "fake" `upstream/master` usinig `git branch -D upstream/master`.

There are outpusts of commands:


`$ git show-ref --head --dereference`

`0e095eaa75f1f12069d67485c5406c19fa123103 HEAD`
`0e095eaa75f1f12069d67485c5406c19fa123103 refs/heads/master`
`2b51f3f1ae4046652cf603bec80778d4722ad51f refs/remotes/origin/HEAD`
`2b51f3f1ae4046652cf603bec80778d4722ad51f refs/remotes/origin/master`
`9945119b571647e6d2f11fa9bd164b798cbd246c refs/remotes/origin/patch-add-editorconfig`
`143eda9f8c6e97e05099c259b63ac9ba0afef00e refs/remotes/origin/remove-main-Kottans-chat-link`
`2b51f3f1ae4046652cf603bec80778d4722ad51f refs/remotes/upstream/master`
`2d69ab0f08efda5e37d1104edb977c1b18f84af1 refs/tags/v.2017-2018`


` git log --graph --full-history --all --color --pretty=format:"%x1b[31m%h%x09%x1b[32m%d%x1b[0m%x20%s" -n10`

`* 0e095ea        (HEAD -> master) say hi`
`*   2b51f3f      (upstream/master, origin/master, origin/HEAD) Merge pull request #172 from kottans/patch-add-editorconfig`
`|\`
`| * 9945119      (origin/patch-add-editorconfig) Add trailing newline :)`
`| * 43fd5f3      Add more defaults to .editorconfig`
`| * 75b1832      Add .editorconfig file`
`* | b5d4716      greeting for kottans (#170)`
`* | 30f2bf5      edit readme.md (#171)`
`|/`
`* 2d56578        Add cat force gif (#169)`
`* 337b901        Add new text in READ (#168)`
`*   1f0a5de      Merge pull request #164 from MitchfFirstGit/master`
`|\`

Tell me, please, what is the next step?

**Comment by ApolloSG** [comment]
Sorry for close/open. Miss click

**Comment by OleksiyRudenko** [comment]
@ApolloSG Ok, great. Topology like it should be, and a tip of `upstream/master` is where it is supposed to be - at a commit that is present in the core repo.
You also have local `master` one commit ahead of `origin/master`, which is also OK.

Make sure you're on your local `master`: `git checkout master`

Let's check if git allows you to push.

`git push origin master`

If git reports error, please, copy/paste your console here.

**Comment by ApolloSG** [comment]
Got the error
`$ git push origin master`
`remote: Permission to kottans/mock-repo.git denied to ApolloSG.`
`fatal: unable to access 'https://github.com/kottans/mock-repo.git/': The requested URL returned error: 403`

**Comment by OleksiyRudenko** [comment]
@ApolloSG what `git remote -v` says?

**Comment by ApolloSG** [comment]
@OleksiyRudenko 

`$ git remote -v`
`origin  https://github.com/kottans/mock-repo.git (fetch)`
`origin  https://github.com/kottans/mock-repo.git (push)`
`upstream        https://github.com/kottans/mock-repo.git (fetch)`
`upstream        https://github.com/kottans/mock-repo.git (push)`

**Comment by OleksiyRudenko** [comment]
@ApolloSG oh, no! )))
You see, the `origin` points at the core repo, where you indeed have no `write` permissions.
Replace `origin` url with your fork url: `git remote set-url origin https://github.com/ApolloSG/mock-repo.git`

Everything should work since.

**Comment by ApolloSG** [comment]
@OleksiyRudenko 

`$ git push origin master`
`To https://github.com/ApolloSG/mock-repo.git`
` ! [rejected]        master -> master (fetch first)`
`error: failed to push some refs to 'https://github.com/ApolloSG/mock-repo.git'`
`hint: Updates were rejected because the remote contains work that you do not have locally. This is usually caused by another repository pushing to the same ref. You may want to first integrate the remote changes (e.g., 'git pull ...') before pushing again. See the 'Note about fast-forwards' in 'git push --help' for details.`

Should i `git pull` first?

**Comment by OleksiyRudenko** [comment]
@ApolloSG Oh, yes! Now you can )

**Comment by OleksiyRudenko** [comment]
@ApolloSG commits look a bit excessive, but this is not really a problem, I shall squash-merge.
My other question: is there your addition in the diff?

**Comment by ApolloSG** [comment]
@OleksiyRudenko 
Most likely no. I just solved the conflict in pulling merge. Perhaps added or removed an empty line somewhere.

**Comment by OleksiyRudenko** [comment]
@ApolloSG we're almost done.
Add your message, commit, then push. Make sure there is an empty line at the end of `README.md`.

**Comment by ApolloSG** [comment]
@OleksiyRudenko 

I see there is no empty line under my message. But i was when i was saving the file.

**Comment by ApolloSG** [comment]
@OleksiyRudenko 

Are we done?

**Comment by OleksiyRudenko** [comment]
@ApolloSG You did a great job with a quite complicated repo conflict case, also you showed notable perseverance.
You deserve special recognition.

Here is your achievement badge:
 [Kottans-Student-Conflict-resolution%3D-mastered%20conflict%20resolution-brightgreen.svg)](https://github.com/kottans/kottans/blob/master/endorsements.md)

Please, provide me with a link to a document in your course repo so I could PR the badge code into.
I need it here for history tracking and proof.

Congratulations on the skills level-up!

**Comment by ApolloSG** [comment]
@OleksiyRudenko 

There is a link https://github.com/ApolloSG/kottans-frontend
Many thanks for the help and patience)

P.S.
[My face](https://media.giphy.com/media/5VKbvrjxpVJCM/giphy.gif) when i was reading about achievementXD

**Comment by OleksiyRudenko** [comment]
@ApolloSG My pleasure. Keep going!
[PR in your repo](https://github.com/ApolloSG/kottans-frontend/pull/1) for your review.

**Pull request merged**